examples/pylint/R0915_too_many_statements.py

```diff
@@ -1,9 +1,11 @@
-def statement(arg):
-    """Dummy function to demonstrate an example within too_many_statements()"""
+from typing import Any
+
+def statement(arg: Any) -> None:
+    """Dummy function to demonstrate an example within too_many_statements()."""
```

Comment:
Removed.

Should I enclose the function name with backticks (`` ` ``) or angle brackets (` < `)?